nodebuilder/settings.go

```diff
@@ -21,10 +34,74 @@ func WithBootstrappers(peers params.Bootstrappers) fx.Option {
 }
 
 // WithMetrics enables metrics exporting for the node.
-func WithMetrics(enable bool) fx.Option {
-	return fx.Options(
-		// TODO: Add metrics to more modules and enable here.
-		headermodule.WithMetrics(enable),
-		daser.WithMetrics(enable),
+func WithMetrics(enable bool, metricOpts []otlpmetrichttp.Option, nodeType node.Type) fx.Option {
```

Comment:
Where is `WithMetrics` called that needs to pass in `false`? Let's remove this param?

Reply:
it just returns fx.Option so no need to short-circuit with the `enable` param

Reply:
done